app/views/layouts/application.html.slim

```diff
@@ -93,3 +93,28 @@ html.no-js lang="en"
           ARTICLE_LIMIT = #{ENV["ARTICLE_LIMIT"].to_i},
           VIZ_LIMIT = #{ENV["VIZ_LIMIT"].to_i};
       #{ "var search_total_found = #{@total_found};" if !@total_found.nil? }
+
+  / Google Tag Manager
```

Comment:
No. Apparently that is now you comment in slim templates
https://github.com/slim-template/slim#code-comment-